Corzbank.Web/ClientApp/src/app/header/wallet/wallet.component.html

```diff
@@ -50,9 +48,112 @@ <h2>Deposits</h2>
 				<hr />
 			</div>
 		</div>
-		<div class="deposits" *ngIf="depositsIsDisplayed"></div>
+		<div class="deposits-block" *ngIf="depositsIsDisplayed">
+			<div class="deposits">
+				<h1 *ngIf="deposits.length === 0" class="none-items">
+					You do not have any deposits
+				</h1>
+				<h1 *ngIf="cards.length === 0" class="none-items">
+					Firstly you need to create a card
+				</h1>
+				<div class="deposit" *ngFor="let deposit of deposits.slice().reverse()"
```

Comment:
Why do you use slice?

Reply:
reverse() doesn't work without slice()

Reply:
![image](https://user-images.githubusercontent.com/44134953/184668081-152331a9-547f-4e4a-9825-cdcc05a05317.png)

Reply:
it doesn't work in my case